src/Core/Models/Api/Request/EmergencyAccessRequstModels.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Bit.Core.Models.Table;
+
+namespace Bit.Core.Models.Api.Request
+{
+    public class EmergencyAccessInviteRequestModel : IValidatableObject
+    {
+        [Required]
+        [EmailAddress]
+        public string Email { get; set; }
+        [Required]
+        public Enums.EmergencyAccessType? Type { get; set; }
+        [Required]
+        public int WaitTimeDays { get; set; }
+
+        public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+        {
+
+            if (Email.Contains(" ") || Email.Contains("<"))
+            {
+                yield return new ValidationResult($"Email is not valid.",
+                    new string[] { nameof(Email) });
+            }
```

Comment:
Should be handled by the EmailAddress attribute you are already using, right?

Reply:
`[EmailAttribute]` seems to only validate
```
// only return true if there is only 1 '@' character
// and it is neither the first nor the last character
```

I don't believe we will have any issues allowing these fields. The Org invite contains these checks which the code is based on.